Makefile

```diff
@@ -12,9 +20,10 @@ ssh:
 	-e "AWS_ACCESS_KEY_ID=${AWS_ACCESS_KEY_ID}" \
 	-e "AWS_SECRET_ACCESS_KEY=${AWS_SECRET_ACCESS_KEY}" \
 	-e "AWS_DEFAULT_REGION=${AWS_DEFAULT_REGION}" \
+	-e MOBILIZE_AMERICA_API_KEY \
+	-e MONGODB_URI \
+	-p "3000:3000" \
```

Comment:
Also let's avoid using port 3000 here since thats what the website uses. Would be nice if both local API's and the website could run at the same time if you had to debug something or work on something experimental.

Reply:
Good point -- will follow up on this next time I hack on this endpoint.